cmd/fyne_demo/main.go

```diff
@@ -88,7 +88,9 @@ func main() {
 		widget.NewTabItemWithIcon("Graphics", theme.DocumentCreateIcon(), screens.GraphicsScreen()),
 		widget.NewTabItemWithIcon("Windows", theme.ViewFullScreenIcon(), screens.DialogScreen(w)),
 		widget.NewTabItemWithIcon("Advanced", theme.SettingsIcon(), screens.AdvancedScreen(w)))
-	tabs.SetTabLocation(widget.TabLocationLeading)
```

Comment:
Should this be removed by this PR? It maybe an unnecessary call as leading is the default location, but this code is mean for showing devs how to use the framework so could be helpful to leave

Reply:
*correction: TabLocationTop is the default, so this line is needed

Reply:
@stuartmscott Sorry, it was deleted by accident. Will revert later.